EVGEN/AliGenPerformance.cxx

```diff
@@ -88,35 +90,35 @@ AliGenPerformance::AliGenPerformance():
 }
 //-----------------------------------------------------------------------------
 AliGenPerformance::AliGenPerformance(const char* generName, Int_t verboseLevel):
-  AliGenerator(),
-  fNJets(1),                // mean number of jets per event
-  fF1Momentum(0),           // momentum distribution function 
-  fFPhi(0),                 // phi distribution function
-  fFTheta(0),               // theta distribution function
-  fFPosition(0),            // position distribution function 
-  fFPdg(0),                 // pdg distribution function  
-  fStreamer(0),           // test stream - used for tuning of parameters of generator
-  fVerboseLevel(0)            // verbose level
+        AliGenerator(),
+        fNJets(1),                // mean number of jets per event
+        fF1Momentum(0),           // momentum distribution function
+        fFPhi(0),                 // phi distribution function
+        fFTheta(0),               // theta distribution function
+        fFPosition(0),            // position distribution function
+        fFPdg(0),                 // pdg distribution function
+        fStreamer(0),           // test stream - used for tuning of parameters of generator
+        fVerboseLevel(0)            // verbose level
```

Comment:
@sawenzel 
Can we move further discussion to JIRA as it is not directly related to pull request?

Before JIRA task will be created, here are my consideration:
I think we agree we need some default formatting. 
For your point a..c they are not easy applicable.

* Currently I'm both users from your example: user  A (CLIon editor) and user B (XEmacs editor)
  * in my commit I used auto-intent of CLion
* To avoid problems with new code I have to know which formatting to use
  * it is not important what formatting we will use, but it is should be consistent
  * preferably "back compatible" in sense - minimal changes in existing code
* For old code my editors are incompatible  - **mostly when tabs are mixed with spaces**
  * Code is in some cases was practically unreadable

For your point c I had to reformat the code (reformatting is not done by default) as I could not read it (I can post screenshot). Once the code was reformatted I could reapply diff with -w option, (as yous suggest?), but I do not thing is is good idea. Code will be again not readable  and this is not what I wanted.
Next time - I will  explicitly mention formatting changes in commit.

In case we can not make a revolution (with old code), I prefer to apply default style for new code.
I need to know which style to apply:
*  K&R  ( as stated in https://en.wikipedia.org/wiki/Indent_style)
  *  most consistent with what we have currently in AliRoot and Root.
* how many spaces ?
  * in AliRoot we have mix of 2 resp. 4 space (sometimes tabs)